pkgs/applications/radio/openwebrx/default.nix

```diff
@@ -82,19 +84,21 @@ let
 in
 buildPythonApplication rec {
   pname = "openwebrx";
-  version = "1.2.2";
+  version = "1.2.82";
 
   src = fetchFromGitHub {
-    owner = "jketterl";
-    repo = pname;
-    rev = version;
-    hash = "sha256-i3Znp5Sxs/KtJazHh2v9/2P+3cEocWB5wIpF7E4pK9s=";
+    owner = "luarvique";
+    repo = "openwebrx";
+    rev = "0e13d5f290430d127cf0accd4e3c29e7435ee645";
+    hash = "sha256-fwU1aMsjMMiq3zffFs8tksnR5bhTdeoeDQTaaLM0p7M=";
   };
 
   propagatedBuildInputs = [
```

Comment:
Use `dependencies` instead of `propagatedBuildInputs`? I'm not a Python person.

In #347566 I noticed that this fork supports even more decoders. Could you please add at least `dablin`, `hamlib` and `multimon-ng` here? They're already available in nixpkgs. To verify you can check OpenWebRX' decoder status page.